src/routes/__layout-root.svelte

```diff
@@ -63,23 +63,25 @@
   import Banners from '$lib/components/banner/banners.svelte';
   import { ErrorBoundary } from '$lib/components/error-boundary';
   import PageTitle from '$lib/holocene/page-title.svelte';
+  import ScrollToTop from '$lib/holocene/scroll-to-top.svelte';
 
   export let user: User;
   export let uiVersionInfo: UiVersionInfo;
 </script>
 
 <PageTitle />
-<div class="flex h-screen w-screen flex-row">
+<div class="flex w-screen flex-row">
```

Comment:
The `h-screen` was messing with the `on:scroll` here, hence it's removal.

Reply:
I'm cool with this, but make sure to link with cloud ui and test it there to see if there's any wonkiness importing it in. That repo also uses several h-screen classes

Reply:
Does `cloud-ui` use this layout? I've also made [updates](https://github.com/temporalio/cloud-ui/pull/247) directly to the layout in `cloud-ui`.